cql/src/main/java/io/stargate/cql/CqlActivator.java

```diff
@@ -99,6 +99,19 @@ private static Config makeConfig() {
       c.rpc_address = listenAddress;
       c.native_transport_port = cqlPort;
 
+      c.native_transport_max_concurrent_connections =
+          Long.getLong("stargate.cql.native_transport_max_concurrent_connections", -1);
+      c.native_transport_max_concurrent_connections_per_ip =
+          Long.getLong("stargate.cql.native_transport_max_concurrent_connections_per_ip", -1);
+      c.native_transport_max_concurrent_requests_in_bytes =
+          Long.getLong(
+              "stargate.cql.native_transport_max_concurrent_requests_in_bytes",
+              Runtime.getRuntime().maxMemory() / 10);
+      c.native_transport_max_concurrent_requests_in_bytes_per_ip =
```

Comment:
what does it mean in our context (per `ip`)? How this is handled?

Reply:
Per client IP I guess..

Reply:
Yeah, it is described in: https://cassandra.apache.org/blog/Improving-Apache-Cassandras-Front-Door-and-Backpressure.html